app/src/androidTest/java/org/mozilla/focus/activity/URLCompletionTest.java

```diff
@@ -231,23 +256,26 @@ private static void removeACSite() {
                 .perform((click()));
     }
 
-    private static void addAutoComplete(String sitename, boolean... checkSuccess) {
-        DataInteraction CustomURLRow = onData(anything())
-                .inAdapterView(allOf(withId(android.R.id.list),
-                        childAtPosition(
-                                withId(android.R.id.list_container),
-                                0)))
-                .atPosition(4);
-        CustomURLRow.perform(click());
+    private void addAutoComplete(String sitename, boolean... checkSuccess) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            CustomURLRow.perform(click());
+        }
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 9 should be on the same line as the next part of a multi\-block statement \(one that directly contains multiple blocks: if/else\-if/else, do/while or try/catch/finally\)\.